src/stores/feedbacks.js

```diff
@@ -35,5 +39,40 @@ export const useFeedbacksStore = defineStore("feedbacks", {
 
       return statusNumbers;
     },
+    //userStore.selectedCategories includes the value added when button is clicked. With code below I check if this value is the same like feedback.category.
+    [INCLUDE_FEEDBACK_BY_CATEGORY]: () => (feedback) => {
+      const userStore = useUserStore();
+      if (userStore.selectedCategories.length === 0) return true;
+      if (userStore.selectedCategories === "All") return true;
```

Comment:
this one will be tricky - I'd put a value here rather than string. f.e === 0 (value: 0, name: All), because you won't have issues with the copy change.

Reply:
done ;)